jscomp/others/js_promise2.ml

```diff
@@ -0,0 +1,16 @@
+include Js_promise
+
+let then_ : 'a promise -> ('a -> 'b promise) -> 'b promise =
```

Comment:
`then_` does not really need the underscore, see

https://github.com/ryyppy/rescript-promise/blob/7eeb332ac2d6749e2b4886aac621658aa888b861/src/Promise.res#L12

Reply:
However, if you only define `then`, you would include the pipe-last `then_` from `Js_promise`. Maybe better to just copy the stuff from `Js_promise` rather than including anyway?